app/assets/stylesheets/bootswatch/_bootswatch.scss

```diff
@@ -0,0 +1,191 @@
+// Sandstone 3.3.2
+// Bootswatch
+// -----------------------------------------------------
+
+@import url("//fonts.googleapis.com/css?family=Roboto:400,500");
+
+// Navbar =====================================================================
+
+.sandstone {
+  font-size: 11px;
+  line-height: 22px;
```

Comment:
Properties should be ordered font-size, font-weight, line-height, text-transform